build.gradle

```diff
@@ -14,9 +14,8 @@ repositories {
     mavenCentral()
 }
 
-def grpcVersion = '1.40.1'
-def protobufVersion = '3.17.3'
-def protocVersion = '3.17.3'
```

Comment:
Removed `protocVersion` and use `protobufVersion` since they should be pointing to the same version.